luigi/rpc.py

```diff
@@ -104,22 +104,26 @@ def __init__(self, url='http://localhost:8082/', connect_timeout=None):
             connect_timeout = config.getfloat('core', 'rpc-connect-timeout', 10.0)
         self._connect_timeout = connect_timeout
 
+        self._rpc_retry_attemps = config.getint('core', 'rpc-retry-attempts', 3)
+        self._rpc_retry_wait = config.getint('core', 'rpc-retry-wait', 30)
+
         if HAS_REQUESTS:
             self._fetcher = RequestsFetcher(requests.Session())
         else:
             self._fetcher = URLLibFetcher()
 
     def _wait(self):
-        time.sleep(30)
+        logger.info("Wait for %d seconds" % self._rpc_retry_wait)
+        time.sleep(self._rpc_retry_wait)
 
-    def _fetch(self, url_suffix, body, log_exceptions=True, attempts=3):
+    def _fetch(self, url_suffix, body, log_exceptions=True):
         full_url = _urljoin(self._url, url_suffix)
         last_exception = None
         attempt = 0
-        while attempt < attempts:
+        while attempt < self._rpc_retry_attemps:
             attempt += 1
             if last_exception:
-                logger.info("Retrying...")
+                logger.info("Retrying %r/%r" %(attempt, self._rpc_retry_attemps))
```

Comment:
Not a human readable error message. Can you come up with something that can be understood on it's own?